ego4d/internal/human_pose/main.py

```diff
@@ -100,11 +100,17 @@ def __init__(self, camera_name, camera_model):
         self.camera_model = camera_model
 
     def image_to_world(self, point_2d):
-        return self.camera_model.projectionModel.unproject(point_2d)[:2]
+        try:
+            return self.camera_model.projectionModel.unproject(point_2d)[:2]
```

Comment:
Since we are all using the newer version now, maybe we could just keep the newer version and comment out the older version just for reference?